docs/api.md

```diff
@@ -1,24 +1,25 @@
-# auro-alerts
+# auro-alert
 
 ## Attributes
 
-| Attribute | Type      | Description                   |
-|-----------|-----------|-------------------------------|
-| `fixed`   | `Boolean` | uses px values instead of rem |
+| Attribute     | Type      | Description                        |
+|---------------|-----------|------------------------------------|
+| `error`       | `Boolean` | Turns alert into error style       |
+| `fixed`       | `Boolean` | uses px values instead of rem      |
+| `information` | `Boolean` | Turns alert into information style |
+| `success`     | `Boolean` | Turns alert into success style     |
+| `warning`     | `Boolean` | Turns alert into warning style     |
 
 ## Properties
 
 | Property         | Attribute        | Type      | Description                                      |
 |------------------|------------------|-----------|--------------------------------------------------|
-| `error`          | `error`          | `Boolean` | Turns alert into error style                     |
 | `hidden`         | `hidden`         | `Boolean` | If present, the component will be hidden both visually and from screen readers |
 | `hiddenAudible`  | `hiddenAudible`  | `Boolean` | If present, the component will be hidden from screen readers, but seen visually |
 | `hiddenVisually` | `hiddenVisually` | `Boolean` | If present, the component will be hidden visually, but still read by screen readers |
-| `information`    | `information`    | `Boolean` | Turns alert into information style               |
 | `noIcon`         | `noIcon`         | `Boolean` | Removes icon from alert UI                       |
 | `role`           | `role`           | `String`  | The role will be set based on type               |
-| `success`        | `success`        | `Boolean` | Turns alert into success style                   |
-| `warning`        | `warning`        | `Boolean` | Turns alert into warning style                   |
+| `type`           | `type`           | `String`  | Component will render visually based on which type value is set; currently supports `error`, `warning`, `success`, `information` |
```

Comment:
How did this get here? I am not seeing an update to the src code to support this. 

I mean, it's correct, but when I run the API build locally, I get 

```
| `type`           | `type`           | `string`  |                                                  |
```

Reply:
I corrected the case of string to String and added the description manually.